### ISSUE TITLE
unable to run lmer // Matrix error

### ISSUE THREAD

**Issue by AnnaAntonatouPap** [lme4/lme4#764]
Hello,

In my new computer were I use Ubuntu 22.04.3 LTS and 4.3.2
I just installed lme4_1.1-35.1 and Matrix_1.6-2 but I am receiving the following error whan I am running my data and also the toy dataset  sleepstudy  meaning(  `fm1 <- lmer(Reaction ~ Days + (1 | Subject), sleepstudy)`  )

```
Error in t(do.call(sparseMatrix, do.call(rbind, lapply(seq_along(blist),  : 
  object 'Csparse_transpose' not found
```

If anyone has any idea how to fix this I will be grateful, it's quite urgent.
I checked a similar issue here but I could figure  it out.

Thanks in advance,
Anna

**Comment by bbolker** [comment]
Start by trying `install.packages("lme4", type = "source")` and see if that helps ...

**Comment by jaganmn** [comment]
The message suggests to me that some loaded namespace caches a stale method for generic function `t`.  *That* package (unlikely **lme4**, whose namespace does not contain an object called `.__T__t:base`) must be reinstalled from sources.  If @bbolker's suggestion does not work, then please show us the output of `sessionInfo()` after you encounter the error.

**Comment by bbolker** [comment]
@jaganmn makes a good point.  Another way to test that hypothesis is to run your example from a completely clean R session (use the `--vanilla` flag if possible &mdash; I don't know how to do that if you're e.g. using RStudio though &mdash; and in any case make sure that your global environment is empty (you haven't loaded any `.RData` files automatically) and only `lme4` and associated packages have been loaded, e.g. for me the end of `sessionInfo()` is

```

attached base packages:
[1] stats     graphics  grDevices datasets  utils     methods   base     

other attached packages:
[1] lme4_1.1-35.1 Matrix_1.6-4 

loaded via a namespace (and not attached):
 [1] minqa_1.2.6    MASS_7.3-60.1  compiler_4.4.0 Rcpp_1.0.11.2  bspm_0.5.3    
 [6] splines_4.4.0  nlme_3.1-163   grid_4.4.0     nloptr_2.0.3   boot_1.3-28.1 
[11] lattice_0.22-5
```

**Comment by AnnaAntonatouPap** [comment]
Hey yes so in a clean environment is working! There is a conflict with phyloseq object but I will try to re-install everything again

**Comment by mmaechler** [comment]
@AnnaAntonatouPap : It may be sufficient to just re-install package `phyloseq` -- either from CRAN (where the binaries are good) *or* using   `install.packages(*, type="source")`

**Comment by DonnerCurves** [comment]
I'd installed the new version of R (4.3.2) and reinstalled lme4 and Matrix but kept getting this error on a call that I'd used many times:  
M_3<- lmer(lnalpha ~ region*period +(1|region:Station) + 
+              (1|period:fYear), 
+              data=mwra, REML=TRUE)
Error in initializePtr() : 
  function 'chm_factor_ldetL2' not provided by package 'Matrix'
I tried reinstalling lme4 and Matrix several times but received the same error.  Following Ben Bolkers 12/14/23 post, I reinstalled with install.packages("lme4", type = "source") and my lmer models runs fine now! Thank you Ben.
